pkg/plugin/v3/scaffolds/init.go

```diff
@@ -84,10 +83,7 @@ func (s *initScaffolder) scaffold() error {
 	bpFile.Path = s.boilerplatePath
 	bpFile.License = s.license
 	bpFile.Owner = s.owner
-	if err := machinery.NewScaffold().Execute(
-		s.newUniverse(""),
-		bpFile,
-	); err != nil {
+	if err := machinery.NewScaffold().Execute(s.newUniverse(""), bpFile); err != nil {
```

Comment:
I would leave this as it was to follow the same pattern as in every other call to `Execute`. I know in this case it is only a single file but it is easy to see the similarities when they follow the same style.